subprojects/soak/src/integTest/kotlin/org/gradle/kotlin/dsl/caching/ScriptCachingIntegrationTest.kt

```diff
@@ -234,19 +243,27 @@ class ScriptCachingIntegrationTest : AbstractScriptCachingIntegrationTest() {
         val settingsFile = cachedSettingsFile(withSettings(""), false, false)
         val buildFile = cachedBuildFile(withBuildScript("""task<MyTask>("myTask")"""), true)
 
+        // and: kotlin-dsl cache assertions
+        fun KotlinDslCacheFixture.assertCacheHits(run: Int) {
+            if (run == 1) {
+                misses(settingsFile.stage1)
+                hits(settingsFile.stage2)
+            } else {
+                hits(settingsFile)
+            }
+            hits(settingsFile.stage2)
```

Comment:
❌ This line can be removed for the condition is already implied by the two branches above.

Reply:
Ah, I missed that thanks, and done